spec/models/metrics/rejection_percentage_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Metrics::RejectionPercentage do
+  Rejection::REASONS.each do |reason|
+    it "##{reason} defaults to 0" do
+      expect(subject.public_send(reason)).to eq(0)
+    end
+  end
+
+  describe '#attributes' do
```

Comment:
No need for more. Perhaps a better name would be clearer.
Instead of ```has all the attributes``` it would be ```has numeric attributes initialised to 0``` ?